.github/workflows/check-frontend.yml

```diff
@@ -0,0 +1,31 @@
+name: Check Frontend
+on: [push]
+
```

Comment:
Please change this so that it runs on PRs + push on master + run only if frontend code is changed. See [backend conf](https://github.com/cs350-team4/ats/blob/master/.github/workflows/check-backend.yml#L4) for example

Reply:
done. also, added readme.